context.tf

```diff
@@ -34,27 +34,16 @@ module "this" {
   label_order         = var.label_order
   regex_replace_chars = var.regex_replace_chars
   id_length_limit     = var.id_length_limit
+  label_key_case      = var.label_key_case
+  label_value_case    = var.label_value_case
 
   context = var.context
 }
 
 # Copy contents of cloudposse/terraform-null-label/variables.tf here
 
 variable "context" {
-  type = object({
-    enabled             = bool
-    namespace           = string
-    environment         = string
-    stage               = string
-    name                = string
-    delimiter           = string
-    attributes          = list(string)
-    tags                = map(string)
-    additional_tag_map  = map(string)
-    regex_replace_chars = string
-    label_order         = list(string)
-    id_length_limit     = number
-  })
```

Comment:
I do not feel comfortable with modifying context.tf since a standard supported across modules.
I think if we need a min_lenght then we should add that to the null-label module to support it.

Reply:
@jamengual I'm a bit confused - this is the latest context.tf that is installed via `make github/init` in the build-harness.

Reply:
ohhhhh sorry, I guess this module has not been updated in a while......

Reply:
No problem! I just rebased onto the latest master and fixed the conflicts.